dispatches/models/fossil_case/properties/hitecsalt_properties.py

```diff
@@ -270,29 +226,19 @@ def initialize(blk, state_args=None,
                 return flags
             else:
                 blk.release_state(flags)
+        init_log.info('Initialization Complete.')
 
-        if outlvl > 0:
-            if outlvl > 0:
-                _log.info('{} Initialisation Complete.'.format(blk.name))
-
-    def release_state(blk, flags, outlvl=0):
+    def release_state(blk, flags, outlvl=idaeslog.NOTSET):
         # Method to release states only if explicitly called
 
+        init_log = idaeslog.getInitLogger(blk.name, outlvl, tag="properties")
```

Comment:
This looks like it should be a global declaration, not part of a method on the class.